app/models/visit.rb

```diff
@@ -19,6 +19,7 @@
 # TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 class Visit < ApplicationRecord
+  attr_accessor :set_all_insurance
```

Comment:
Instead of adding the `attr_accessor`, you can make the check box tag just `= check_box_tag` instead of attaching it to the form, that way it just comes across as `params[:set_all_insurance]` in the controller